docs/ClientApiInstrucitons.md

```diff
@@ -0,0 +1,18 @@
+# Client API Setup and Running
+
+*Reference: `https://angular.io/guide/http` for any additional clarifications on Angular's http client object.*
+
+## DEVELOPMENT REFERENCE
+* To Use the Client API 
+  * Include the Api service: `import { ServerApiService } from '../services/ServerApi.service';`
+  * Add the Service to the Component Constructor: `constructor (private appointments: ServerApiService) { }`
+  * Call the Service and add the appropriate `.subscribe()` function: For our own convetion: Expected objects will be placed in: `src/app/assets/ApiObjects` for typechecking purposes.
+    * **Example**: Code to be placed in the component 
+        ```javascript
+            this.serverApiService.getTest().subscribe( (response: EXPECTED_OBJECT_TYPE) => {
```

Comment:
Good Question that I didn't clarify: It should be defined in the ServerApiService.

Reply:
I'll push up a separate DUMMY example API branch to demonstrate what this document is attempting to describe.
Because I COMPLETELY left out how to handle how to implement an new public Service Function.

Reply:
On consideration of creating a DUMMY example. the main issue with this implementation is the violation of the one-per-component rule that Angular's style guide suggests